components/tidb_query/src/codec/mysql/decimal.rs

```diff
@@ -2821,25 +2823,25 @@ mod tests {
             (WORD_BUF_LEN, b"2.23E2abc", Res::Ok("223")),
             (WORD_BUF_LEN, b"2.23a2", Res::Ok("2.23")),
             (WORD_BUF_LEN, b"223\xE0\x80\x80", Res::Ok("223")),
-            (WORD_BUF_LEN, b"1e -1",Res::Ok("0.1")),
-            (WORD_BUF_LEN, b"1e001",Res::Ok("10")),
+            (WORD_BUF_LEN, b"1e -1", Res::Ok("0.1")),
```

Comment:
Please remove the indents. In https://github.com/tikv/tikv/pull/5255, it adds `#[rustfmt::skip]` to this test